app/src/main/java/de/mwvb/blockpuzzle/SelectLanguage.java

```diff
@@ -0,0 +1,66 @@
+package de.mwvb.blockpuzzle;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.annotation.SuppressLint;
+import android.content.Context;
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.content.res.Configuration;
+import android.content.res.Resources;
+import android.os.Bundle;
+import android.preference.PreferenceManager;
+import android.view.View;
+import android.widget.Button;
+import android.widget.ImageButton;
+
+import java.util.Locale;
+
+import de.mwvb.blockpuzzle.logic.Game;
+
+public class SelectLanguage extends AppCompatActivity {
+
+    boolean lang_selected = true;
+    Context context;
+    Resources resources;
+    final  String[] language = {"english","german"};
+
```

Comment:
Please try to avoid writing more than one blank line. My architect told me at the time: "There is no reason for that"